get_npm.js

```diff
@@ -4,8 +4,9 @@ var util = require('util'),
     wget = require('./wget');
 
 var NPM_PKG_JSON_URL = 'https://raw.githubusercontent.com/joyent/node/%s/deps/npm/package.json';
-var NVMW_NODEJS_ORG_MIRROR = process.env.NVMW_NODEJS_ORG_MIRROR || 'http://nodejs.org/dist';
-var BASE_URL = NVMW_NODEJS_ORG_MIRROR + '/npm/npm-%s.zip';
+// https://github.com/npm/npm/tags
+var NVMW_NPM_MIRROR = process.env.NVMW_NPM_MIRROR || 'https://github.com/npm/npm/archive';
+var BASE_URL = NVMW_NPM_MIRROR + '/v%s.zip';
 
```

Comment:
auto create `npm.cmd` script is include in https://github.com/deepak1556/nvmw/commit/9fb369337d063fc3dcdbe36f8474f52256394d5c